### PR TITLE
update templates and add some flash scripts

### LINKED ISSUE #5290: Template fixes after Review
Go to Politics, world, multiple choice:
Who will be [Position] of [Location] by [Specific Datetime]?

1) Position drop down ... Change Prince to Crown Prince

2) Also need to add an additional Rule to this market under  Added Resolution Rules saying:

If the Country in this market has multiple people with the same title, the market should resolve as invalid.

Crypto Market (need to add a sentence into all crypto markets using trading view in its templates)

Add:  To do this you would go to 00:00 on the date specified in the market question and depending on where you are located, you count backwards or forwards the number of candlesticks your time zone is.  For example if you are located in UTC -5 and you wanted to find the Open Price for December 16th on Coinbase, you would go to the hourly chart and search for 00:00 on December 16th and count 5 candlesticks backwards to December 15th 1900 EST.

*****Validate timezones to exchanges for Event resolution times

### PR DESCRIPTION
**Pull request by bthaile**
fixes https://github.com/AugurProject/augur/issues/5290

also added a flash script for creating zerox orders and convenience script for creating markets.

### CODE REVIEW

File: packages/augur-tools/src/templates-lists.ts [hunk 1]
```diff
@@ -235,8 +235,8 @@ export const LIST_VALUES = {
       label: 'King',
     },
     {
-      value: 'Prince',
-      label: 'Prince',
+      value: 'Crowned Prince',
+      label: 'Crowne Prince',
```
Comment (phoebemirman) [inline packages/augur-tools/src/templates-lists.ts]: typo here

**Review by phoebemirman — CHANGES_REQUESTED** [review]
typo - missing a d

File: packages/augur-tools/src/templates-lists.ts (hunk 1 above)
Comment (bthaile) [reply packages/augur-tools/src/templates-lists.ts]: fixed typo

**Comment by bthaile** [comment]
I looked it up `Crown` price seems to be the correct term.

**Comment by bthaile** [comment]
<https://en.wikipedia.org/wiki/Crown_prince> it's a title `crown prince` not the action of crowning or been crowned.

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**